examples/ingest/airtable/ingest.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Processes all the documents in all bases (and all workspaces) within an Airtable org,
+# using the `unstructured` library.
+
+# Structured outputs are stored in airtable-ingest-output
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+cd "$SCRIPT_DIR"/../../.. || exit 1
+
+# Obtain your personal access token, save/source them from another file, for security reasons:
+# source "./../../secrets/airtable.txt"
+# ...
+# --airtable-personal-access-token "$AIRTABLE_PERSONAL_ACCESS_TOKEN"
+
+# Other arguments that you can use:
+# --list-of-paths
+#     --> A list of paths that specify the set of specific locations to ingest data from.
+#     --list-of-paths: path1 path2 path3 ….
+#     path: base_id/table_id(optional)/view_id(optional)/
```

Comment:
is it obvious on the airtable-side, how someone obtains these various ids? any docs we can point them to.

Reply:
Added a short help comment on this, and additional links to documentation 👍
https://github.com/Unstructured-IO/unstructured/blob/22fb177e1df36dea952db304fd057104b17ef363/examples/ingest/airtable/ingest.sh#L22-L34

https://github.com/Unstructured-IO/unstructured/blob/22fb177e1df36dea952db304fd057104b17ef363/unstructured/ingest/cli/cmds/airtable.py#L33-L45